### DISCUSSION TITLE
Welcome to logicola Discussions!

### DISCUSSION THREAD

**Discussion by malikpiara** [malikpiara/logicola#19] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by a13g24** [comment]
Just wanted to say thank you for preserving Gensler's legacy. I wasn't aware he had passed until I came across this repo. Just wanted to say I appreciate your efforts.
